dotcom-rendering/src/web/components/ReaderRevenueLinks.importable.tsx

```diff
@@ -331,6 +331,15 @@ const ReaderRevenueLinksNative: React.FC<{
 			Subscribe <ArrowRightIcon />
 		</a>
 	);
+	const SupportButton = () => (
+		<a
+			css={linkStyles}
+			href={getUrl('contribute')}
```

Comment:
Same comment [as in `frontend`](https://github.com/guardian/frontend/pull/25647#discussion_r1016715259): is this the right URL? I’m assuming we might want a distinct one?

Reply:
No, the URL is to stay the same, please see my reply to your comment in `frontend`.